lib/cli.ex

```diff
@@ -4,11 +4,15 @@ defmodule Onigumo.CLI do
   }
 
   def main(argv) do
-    case OptionParser.parse(argv, strict: []) do
-      {[], [component], []} ->
+    case OptionParser.parse(
+           argv,
+           aliases: [C: :working_dir],
+           strict: [working_dir: :string]
+         ) do
+      {parsed_switches, [component], []} ->
         {:ok, module} = Map.fetch(@components, String.to_atom(component))
-        root_path = File.cwd!()
-        module.main(root_path)
+        working_dir = Keyword.get(parsed_switches, :working_dir, File.cwd!())
```

Comment:
What happens if there is no value passed to the switch? Like

```
./onigumo downloader --working-dir
```

Reply:
Test added.